Quadriga/src/main/java/edu/asu/spring/quadriga/service/textfile/ITextFileManager.java

```diff
@@ -49,4 +50,25 @@
 
     ITextFile getTextFile(String textId) throws QuadrigaStorageException;
 
+    /**
+     * saves or updates the txtFile in the database.
+     * 
+     * @param txtFile
+     *            Text File object from web controller to handle text
+     *            operations.
+     * @return
+     */
+    void saveTextFileToDB(ITextFile txtFile);
```

Comment:
this is a manager class. method names should not have anything about DBs in them.